src/ResourceManager/Sql/Commands.Sql/help/Get-AzureRmSqlServerDnsAlias.md

```diff
@@ -0,0 +1,154 @@
+---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
+Module Name: AzureRM.Sql
+online version: 
+schema: 2.0.0
+---
+
+# Get-AzureRmSqlServerDnsAlias
+
+## SYNOPSIS
+Gets or lists Azure SQL Server DNS Alias.
+
+## SYNTAX
+
+```
+Get-AzureRmSqlServerDnsAlias [-DnsAliasName <String>] -ServerName <String> [-ResourceGroupName] <String>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+```
+
+## DESCRIPTION
+Get the specific Azure SQL Server DNS Alias or lists all Server DNS Aliases for the server
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> $serverDNSAliases = Get-AzureRmSqlServerDNSAlias -ServerName servername -ResourceGroupName rgname
+
+ResourceGroupName  ServerName   DnsAliasName
+-----------------  ----------   ------------------
+rgname             servername   dnsaliasname
+rgname             servername   dnsaliasname2
+```
+
+Lists all Server DNS Aliases for the specific server
+
+### Example 2
+```
+PS C:\> $serverDNSAliases = Get-AzureRmSqlServerDNSAlias -DnsAliasName dnsaliasname -ServerName servername -ResourceGroupName rgname
+
+ResourceGroupName  ServerName   DnsAliasName
+-----------------  ----------   ------------------
+rgname             servername   dnsaliasname
+```
+
+Gets Server DNS Alias specified by server and alias name
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with azure.
+
+```yaml
+Type: IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DnsAliasName
```

Comment:
@maboja-msft looks like the help needs to be updated to reflect the changes that were made in the previous commit(s)